openapi/platform/platform.yaml

```diff
@@ -403,6 +403,122 @@ paths:
               properties:
                 data:
                   $ref: '#/components/schemas/businessLocations'
+  /businessLocationReviews:
```

Comment:
If this is going to be a Reputation API endpoint, I think it should be written within its own yaml file, placed under its own folder like `openapi > reputation > reputation.yaml`.  That is the pattern I used for Ad Intel at least.